### DISCUSSION TITLE
Which values will the plugin get from ista?

### DISCUSSION THREAD

**Discussion by Thomas-W72** [Ludy87/ecotrend-ista#119] [General]
I am new to HA and also do not have access to ista service ecotrend, but the house community intends to use the service. So I would like to know, if the HA plugin gets only the monthly values from ista, which ista will also show on their service, or will the plugin get daily values? It would be great, if the plugin gets daily values, because i would like to have more timely data. You have to set something at ista for the daily data?

Thanks for responses in advance.

**Comment by Ludy87** [comment]
The data provided by Ista is monthly data; Ista does not plan to record it on a daily basis.
